### PR TITLE
ports heart disease from bee

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
quote from orignal pr with one modification changing level 9 to 8 and other things that make it better for the context of our server About The Pull Request
Adds a new simplemob, a really angry heart

"Adds a new level 8 symptom, Heart disease. The only level 8 kill symptom yet, but also quite hard to come back from.

More in depth:
Heart disease has -6 stage speed and a .5% chance, per tick, at stage 5, of causing a heart attack. Unless the stealth 2 threshold is met, this disease is incredibly noticeable as well, and does not debilitate the player until they reach this threshold, making a disease with this symptom easier to counteract if noticed"

Changelog
🆑
add: Heart Disease symptom
add: angry heart simplemob

/🆑

### CODE REVIEW

**Comment by boodaliboo** [comment]
ugh guess im fixing this later

File: code/datums/diseases/advance/symptoms/heartattack.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	resistance = 1
+	stage_speed = -6
+	transmittable = -2
+	level = 8
+	severity = 6
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	var/heartattack = FALSE 
+	threshold_desc = "<b>Transmission 10:</b> When the victim has a heart attack, their heart will pop right out of their chest, and attack!.<br>\
+					  <b>Stealth 2:</b> The disease is somewhat less noticable to the host."
```
Comment (missatessatessy) [inline code/datums/diseases/advance/symptoms/heartattack.dm:27]: threshold_descs = list(
		"Transmission 10" = "When the victim has a heart attack, their heart will pop right out of their chest, and attack!",
		"Stealth 2" = "The disease is somewhat less noticable to the host",
	)

**Comment by nesisknesis** [comment]
God bee are retarded

the transmission buff is autistic

the stealth 2 buff is literally inherent due to the virus having a stat of stealth 2 already. spazcoders.

Heart attacks are shitty

If this were added id put it all my viruses since itd be completely stealth, and would also buff the stage speed so that it almost guarantees my victims death. 

which is why this pr is bad

**Comment by Hopekz** [comment]
That probability per tick isn't too good IMO especially something that kills the user pretty quick after it activates.
A prob 1 per tick seems to activate once every 8 seconds (for example the smoking message on cigarettes) 

So basically with the current balance of this PR at stage -6 you already have a good chance to die in the next 15-20 seconds.

I don't like this PR not sure why you chose to port this out of all other things.
Even with tick balancing to make it less ridiculous I don't really want more diseases that just straight up take you out of the game until you're cloned.
-1

**Comment by boodaliboo** [comment]
It’s a level 8 symptom by the time you get this the rounds a hour in and there is rng anyways on even getting this I will do some nerfs though

**Comment by ghost** [comment]
There is already a symptom that effectively causes this. Bloat.

File: code/datums/diseases/advance/symptoms/heartattack.dm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	small transmission penalty
+	max level
+Bonus
+	heart attacks
+//////////////////////////////////////
+*/
+
+/datum/symptom/heartattack
+	name = "Heart Disease"
+	desc = "This disease infiltrates the host's heart, causing cardiac arrest after a long incubation period"
```
Comment (Ktlwjec1) [inline code/datums/diseases/advance/symptoms/heartattack.dm:16]: full stop

File: code/datums/diseases/advance/symptoms/heartattack.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+/datum/symptom/heartattack/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	if(suppress_warning && M.can_heartattack())
+		if(prob(2)) 
+			to_chat(M, "<span class='warning'>[pick("Your chest aches.", "You need to sit down.", "You feel out of breath.")]</span>")
+	else if(prob(2) && M.can_heartattack())
+		to_chat(M, "<span class='userdanger'>[pick("Your chest hurts!.", "You feel like your heart skipped a beat!")]</span>")
```
Comment (Ktlwjec1) [inline code/datums/diseases/advance/symptoms/heartattack.dm:45]: wtf is this -> !.

File: code/modules/mob/living/simple_animal/hostile/heart.dm [hunk 4]
```diff
@@ -0,0 +1,35 @@
+/mob/living/simple_animal/hostile/heart
+	name = "Heart"
+	desc = "A living heart. It's angry!."
```
Comment (Ktlwjec1) [inline code/modules/mob/living/simple_animal/hostile/heart.dm:3]: !.

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/heart.dm [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+	response_harm = "breaks"
+	density = FALSE
+	attacktext = "beats"
+	ventcrawler = VENTCRAWLER_ALWAYS
+	attack_sound = 'sound/effects/singlebeat.ogg'
+	stat_attack = UNCONSCIOUS
+	attack_same = 1
+	gold_core_spawnable = HOSTILE_SPAWN
+	see_in_dark = 8
+	deathmessage = "falls lifeless."
```
Comment (Ktlwjec1) [inline code/modules/mob/living/simple_animal/hostile/heart.dm:30]: stops beating.

Just a suggestion, ignore if you want.

**Comment by CMOisLing** [comment]
> It’s a level 8 symptom by the time you get this the rounds a hour in and there is rng anyways on even getting this I will do some nerfs though

More like 15 to 30 depending on mining

Im not a big fan of the concept of this symptom, with high stage speed its just a luck game on whether you can walk to medbay before you get insta KO'ed and die shortly after. Combined with some annoying stun symptoms this would be awful. Even if you make it to medbay yogstations finest would probably just shove you in cryo. The whole transmission aspect is completely bypassed by sneezing. We have enough symptoms that are meant to kill you as fast as possible.

**Comment by boodaliboo** [comment]
fine

**Comment by Potototatoh** [comment]
pffffffftgkhpythtohppghppfpgpgphphpttt

**Pull request closed without merging**